src/components/Input/styled.js

```diff
@@ -0,0 +1,86 @@
+import styled from 'styled-components';
+
+export const WrapInput = styled.span `
+    display: flex;
+`;
+
+export const Container = styled.div `
+    position: relative;
+`;
+
+export const Input = styled.input `
+    height: 48px;
+    width: ${({$width}) => $width && $width};
+    margin-bottom: ${({marginBottom}) => marginBottom && marginBottom};
```

Comment:
những cái này nếu em k truyển prop đó vào thì nó thành undefine, 
```suggestion
    ${({$width}) => $width && css`
      width: ${$width};
    `};
    margin-bottom: ${({marginBottom}) => marginBottom && marginBottom};
```